.github/workflows/ci.yaml

```diff
@@ -36,6 +36,7 @@ jobs:
       working-directory: example
       run: flutter pub get
 
+    - run: echo API_KEY=${{ secrets.TEST_API_KEY }} > example/.env
```

Comment:
not needed?

Reply:
we need this since the example app needs this file to be provided in the assets.